### PR TITLE
Created lab results entry.

### PR DESCRIPTION
**Pull request by ynurmahomed**

### CODE REVIEW

**Comment by ynurmahomed** [comment]
[9.2.1. Develop a module to allow the entry of lab test results through the POC interface](https://trello.com/c/ElOKAERd/31-921-develop-a-module-to-allow-the-entry-of-lab-test-results-through-the-poc-interface)

**Comment by valvijo** [comment]
Test results inserted in POC
image
are not aligned with retrospective:
image
This is because `obs` for those tests are not being grouped properly:
image
E.g. There are several `obs_group` for the same concept group `HEMOGRAM KX21N (1723)` but there should be just one. Also wondering why each obs group have a different order_id

File: app/lab/components/testOrderResultItemForm.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<form class="item-form" name="form" ng-submit="vm.onSubmit({item: vm.item, result: vm.$item.value});">
+
+  <div class="form-group" ng-class="{'has-error': vm.hasFocus && form.$dirty && !form.$valid}">
+    <label class="col-sm-4" for="value">{{vm.$item.testOrder.concept.display}}</label>
```
Comment (valvijo) [inline app/lab/components/testOrderResultItemForm.html]: Should display the same labels used in [RDE](http://dev.esaude.org/openmrs/module/htmlformentry/htmlForm.form?id=8), a couple of options:
(1) Map the concepts with custom labels in POC i18n (Preferable)
(2) Rename the concept to use the same RDE form label

File: app/lab/components/testOrderResultList.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  <tbody>
+  <tr ng-repeat="t in vm.testOrderResults" ng-class="{'active': t.selected}">
+    <td>{{t.id}}</td>
+    <td>{{t.dateCreation | date}}</td>
+    <td>{{t.provider.display}}</td>
+    <td>{{t.status}}</td>
+    <td>
+      <button class="btn btn-lg btn-default" type="button" ng-click="vm.onSelectTestOrderResult({testOrderResult: t})">
+        <i class="glyphicon glyphicon-eye-open"></i> {{ 'VIEW' | translate}}
+      </button>
```
Comment (valvijo) [inline app/lab/components/testOrderResultList.html]: Not sure what this button is used for, what other details are available besides those already in the display?

Comment (ynurmahomed) [reply app/lab/components/testOrderResultList.html]: Its used to select the test order when there are more than 1. The first test order is automatically selected.

**Comment by valvijo** [comment]
With  commit [feed936](https://github.com/esaude/openmrs-module-poc/pull/7/commits/feed9364b4d70fe1faeefeb50ec05eea6e20ce25) of `openmrs-module-poc`. POC and RDE lab results are finally aligned. Well done.
image

**Comment by ynurmahomed** [comment]
@valvijo changes implemented.

**Comment by valvijo** [comment]
Some test results are not displaying in RDE after saved in POC:
image

See:
image
and
image

**Comment by ynurmahomed** [comment]
@valvijo I don't think this is something that can solved in the front-end.

**Comment by valvijo** [comment]
The first time I open this screen labels were fine, then I prescribed more tests, and the second time labels are like this:
image

**Comment by ynurmahomed** [comment]
Labels are only translated when using pt locale and also only for tests that are available on RDE.

**Comment by ynurmahomed** [comment]
Maybe something failed. Anything on console?

**Comment by valvijo** [comment]
This is because I changed the language in RDE I was using in the same POC session to En. Back to Pt everything works fine

**Comment by valvijo** [comment]
Looks great!! In case there is any other issues I'll leave that for testers to find.

**Comment by ynurmahomed** [comment]
[Develop POC module](https://trello.com/c/e3nzFiWy/57-develop-poc-module)

**Pull request merged**